crawl-ref/source/dat/descript/backgrounds.txt

```diff
@@ -109,8 +110,8 @@ fighters (especially when unarmed) at the cost of armour.
 %%%%
 Alchemist
 
-Alchemists start with the Sting spell, and their magic is very effective against
-foes susceptible to poison.
+Alchemists start with the Sting spell, and their wide range of effects
+can harm a variety of foes.
```

Comment:
While Alchemists do have very different endspells for a longer while than most starting books, "harm a variety of foes" reads very weirdly when nearly all starting damage spells can hurt most early game foes. I'm not sure what uses of words like "condition", "circumstances", and other such words can properly emphasize their specific limitations to both rPois and the risks of Sticky Flame, but it'd probably be better than going from mentioning they're one of the most conditional ones to vaguely implying they can hurt many things.